src/docs/Components/Hero/index.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Hero'
+order: 1
+description: 'A hero is the first thing your user sees when the page loads. This is a zone that you can use to draw the  attention of the user on a specific message.'
+status:
+  sketch: 'bêta'
+  scss: 'bêta'
+---
+
+> A hero is the first thing your user sees when the page loads. This is a zone that you can use to draw the attention of the user on a specific message.
+
+## Variations
+
+We want to provide as many cases as possible so we created three different versions of heroes.
+
+### Title only hero
+
+This is the most minimal hero case you will find in Mozaïc Design System. This version is best used when you need to introduce a page with just an image and a title on top of it.
+
+--title only preview--
+
+### Title with text hero
```

Comment:
Maybe we can merge this part this the previous one, and indicate that the text is optional.

Reply:
I think that I can be useful to distinguish these two cases